### PR TITLE
Fix onViewableItemsChanged item id

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Show the full image when scrolling in a channel / thread

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53117

```release-note
NONE
```

### CODE REVIEW

**Review by streamer45 — APPROVED** [review]

**Review by lieut-data — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "mattermost-mobile",
-      "version": "2.4.0",
+      "version": "2.5.0",
```
Comment (lieut-data) [inline package-lock.json:9]: Unrelated changes that just arose from an `npm install`, I presume?

Comment (enahum) [reply package-lock.json:9]: yes, 100% accurate your assumption
